### ISSUE TITLE
Depmod complains of read-only filesystem

### ISSUE THREAD

**Issue by LRitzdorf** [VannTen/kernel-modules-hook#3]
While the fix in #2 does cause this project to work as intended, this partially breaks depmod (via the `60-depmod` hook, with description `Updating module dependencies...`).

This appears to be an issue because the hook triggers on any operation (install, upgrade, remove) which touches `usr/lib/modules`, and runs depmod for any/all relevant kernels that have a `modules.order` file. This includes our running kernel, but when depmod runs, it encounters our read-only mounted directory.

This should be fixable if we mount without the read-only option when restoring kernel modules, but I don't know enough about upgrades to tell whether this will cause other issues.

Fixing this is probably out-of-scope for #2, but should be performed in parallel with it.

Relevant Pacman error messages:
```
( 8/12) Updating module dependencies...
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.dep.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.dep.bin.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.alias.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.alias.bin.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.softdep.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.symbols.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.symbols.bin.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.builtin.bin.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.builtin.alias.bin.10071.652225.1706902918, 301, 644): Read-only file system
depmod: ERROR: openat(/lib/modules/6.7.2-arch1-2, modules.devname.10071.652225.1706902918, 301, 644): Read-only file system
```
(6.7.2 is the old, and currently running, kernel.)
